package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dirname-filename-esm",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Please do not change version. It should be changed by `npm version` by maintainer.

Reply:
Ok, I reverted the change.